lib/components/viewers/styled.js

```diff
@@ -41,7 +46,7 @@ export const PatternContainer = styled.div`
 }
 `
 
-export const StopContainer = styled.div`
+export const StopContainer = styled.ul`
```

Comment:
Nit: Do you think the containing list should be an `<ol>` instead of `<ul>`? The pattern viewer does show the stops in the order they are reached.